modules/nw-sriov-hwol-about-hardware-offloading.adoc

```diff
@@ -26,4 +26,6 @@ Only the following two communication types are supported:
 In all cases, hardware offloading takes place only when those pods and services are assigned to nodes that have a compatible SmartNIC.
 Suppose, for example, that a pod on a node with hardware offloading tries to communicate with a service on a regular node.
 On the regular node, all the processing takes place in the kernel, so the overall performance of the pod-to-service communication is limited to the maximum performance of that regular node.
-Hardware offloading is not compatible with DPDK applications.
\ No newline at end of file
+Hardware offloading is not compatible with DPDK applications.
+
+If hardware offloading is enabled on a node but pods running on that node are not configured to use hardware offloading, then pods can have less throughput performance on traffic. You can’t configure hardware offloading for pods that are managed by {product-title}.
```

Comment:
Suggest to check if this can be change to active voice. Suggestion from my end
```suggestion
Enabling hardware offloading on a node, but not configuring pods to use it can result in decreased throughput performance for pod traffic. You can’t configure hardware offloading for pods that are managed by {product-title}.
```